pkg/cmd/generate-postgres-metadata-tables/main.go

```diff
@@ -28,17 +28,36 @@ import (
 	"os"
 
 	"github.com/cockroachdb/cockroach/pkg/sql"
+	"github.com/cockroachdb/cockroach/pkg/sql/types"
 	"github.com/jackc/pgx"
+	"github.com/lib/pq/oid"
 )
 
 const getServerVersion = `SELECT current_setting('server_version');`
 
+// Flags or command line arguments.
 var (
 	postgresAddr   = flag.String("addr", "localhost:5432", "Postgres server address")
 	postgresUser   = flag.String("user", "postgres", "Postgres user")
 	postgresSchema = flag.String("catalog", "pg_catalog", "Catalog or namespace, default: pg_catalog")
 )
 
+// Map for unimplemented types.
+var unimplementedEquivalencies = map[oid.Oid]oid.Oid {
+	// These types only exists in information_schema
+	oid.Oid(13438):     oid.T_int4,        // cardinal_number
+	oid.Oid(13450):     oid.T_bool,        // yes_or_no
```

Comment:
i think you're right -- if we have already started using `text` for `yes_or_no`, let's keep it that way

Reply:
Done.